components/dbt/actions/trigger-job-run/trigger-job-run.mjs

```diff
@@ -0,0 +1,66 @@
+import dbt from "../../dbt.app.mjs";
+
+export default {
+  key: "dbt-trigger-job-run",
+  name: "Trigger Job Run",
+  description: "Trigger a specified job to begin running. [See the documentation](https://docs.getdbt.com/dbt-cloud/api-v2#/operations/Trigger%20Job%20Run)",
+  version: "0.0.1",
+  type: "action",
+  props: {
+    dbt,
+    accountId: {
+      propDefinition: [
+        dbt,
+        "accountId",
+      ],
+    },
+    projectId: {
+      propDefinition: [
+        dbt,
+        "projectId",
+        (c) => ({
+          accountId: c.accountId,
+        }),
+      ],
+      optional: true,
+    },
+    environmentId: {
+      propDefinition: [
+        dbt,
+        "environmentId",
+        (c) => ({
+          accountId: c.accountId,
+          projectId: c.projectId,
+        }),
+      ],
+      optional: true,
+    },
+    jobId: {
+      propDefinition: [
+        dbt,
+        "jobId",
+        (c) => ({
+          accountId: c.accountId,
+          projectId: c.projectId,
+          environmentId: c.environmentId,
+        }),
+      ],
+    },
+  },
+  async run({ $ }) {
+    const { data } = await this.dbt.triggerJobRun({
+      accountId: this.accountId,
+      jobId: this.jobId,
+      data: {
+        cause: "Triggered manually",
```

Comment:
I would be in favor of specifying a different cause, since `Triggered manually` is the same as what shows when triggering jobs from the DBT Cloud UI:
![image](https://github.com/PipedreamHQ/pipedream/assets/66686238/b48a62ca-d2ea-4f3e-963b-a04666004802)

Maybe `Triggered via Pipedream`?

Reply:
Here's how it looks when triggered from our Node.js code steps:
![image](https://github.com/PipedreamHQ/pipedream/assets/66686238/ee586be0-8c8e-4f88-a7fd-620b17f4f83e)

Reply:
Also, maybe it could be an optional prop for the action, in case anyone wants to modify the default value?